src/views/FindPurchaseOrder.vue

```diff
@@ -228,6 +227,14 @@ export default {
 </script>
 
 <style scoped>
+.items-metadata {
+  text-align: end;
+}
+
+.items-metadata :first-child {
```

Comment:
why just the first child?

Reply:
I wanted my first note to take the full width available and push another note in the next line, that's why I had applied on the first note only. There was no need to apply a display block to another note. 
But now I have applied it to both of them.